VirtualLibrary/Forms/Opening.cs

```diff
@@ -20,10 +20,10 @@ private void Label2_Click(object sender, EventArgs e)
 
         private void RegistrationButton_Click(object sender, EventArgs e)
         {
-            
-           new Registration(_mUserRepository).ShowDialog();
-            this.Close();
 
+            this.Hide();
+            new Registration(_mUserRepository).ShowDialog();
+            this.Show();
```

Comment:
I guess it would be better to open a new form of opening because in registration form you choose language and if the old opening form is shown then the language doesn't change.

Reply:
You are right, but shouldn't language select be in opening form (it is first screen user will see) ?

Reply:
I agree. @viktorija-kondratjevaite ?:)